src/net/SDP/SDP.cs

```diff
@@ -309,13 +309,27 @@ public static SDP ParseSDPDescription(string sdpDescription)
                                 break;
 
                             case var l when l.StartsWith("m="):
-                                Match mediaMatch = Regex.Match(sdpLineTrimmed.Substring(2), @"(?<type>\w+)\s+(?<port>\d+)\s+(?<transport>\S+)(\s*)(?<formats>.*)$");
+                                Match mediaMatch = Regex.Match(

```

Comment:
Thanks for fixing this bug! It would be great if it was in a separate PR though to make the review and merge easier and safer.